main.c

```diff
@@ -199,22 +201,68 @@ void updateArm()
 void raiseCarrier()
 {
 	//move carrier motors at speed so it should difinitely be able to lift the mobile goal
-	motor[carrier] = 100;
+	//for(int coefficient = 1; coefficient <= 10; coefficient++)
+	//{
+	//	motor[carrierLeft] = 10*coefficient;
+	//	motor[carrierRight] = 10*coefficient;
+	//	sleep(100);
+	//}
+	for(int base = 1; base <= 10; base++)
+	{
+		motor[carrierLeft] = pow(base,2);
+		motor[carrierRight] = -pow(base,2);
+		sleep(186);
+	}
 	//wait 1 second, needs to be tested and probably changed but this is the test value
-	sleep(1000);
+	//sleep(1000);
 	//stop carrier motors
-	motor[carrier] = 0;
+	motor[carrierLeft] = 0;
+	motor[carrierRight] = 0;
 }
 
+//lowers the mobile goal, needs to be heavily tested
 //lowers the mobile goal, needs to be heavily tested
 void lowerCarrier()
 {
+	//for(int coefficient = 1; coefficient <= 4; coefficient++)
```

Comment:
Same as above- remove if it's not being used

Reply:
On it